### ISSUE TITLE
vgo: mix of old-/new- import paths in `v1.3.0`

### ISSUE THREAD

**Issue by sbinet** [nanomsg/mangos#327]
hi,

the latest tagged version of `mangos` contains a mix of old and new import paths.
this confuses `vgo`:

```
$> pwd
$GOPATH/src/github.com/pouet/pkg

$> cat pkg.go
package pkg // import "github.com/pouet/pkg"

import (
	_ "nanomsg.org/go-mangos/protocol/pair"
)


$> touch go.mod
$> vgo install -v .
vgo: resolving import "nanomsg.org/go-mangos/protocol/pair"
vgo: finding nanomsg.org/go-mangos (latest)
vgo: adding nanomsg.org/go-mangos v1.3.0
vgo: resolving import "github.com/go-mangos/mangos"
vgo: finding github.com/go-mangos/mangos (latest)
vgo: adding github.com/go-mangos/mangos v1.3.0
```

and that's because, indeed, at `v1.3.0`, `protocol/pair/pair.go` contains:
```go
[...]

// Package pair implements the PAIR protocol.  This protocol is a 1:1
// peering protocol.
package pair

import (
	"sync"
	"time"

	"github.com/go-mangos/mangos"
)

```

could a `v1.3.1` be cut with the last remnants of the old import path removed ? (ie: c289f0f would be a great candidate for such a tag.)

**Comment by gdamore** [comment]
Ok, this should be addressed now -- check out v1.4.0.  (And I also moved the v1 tag).

If that doesn't work for you, please let me know.

I'm now wondering if I should insert a canonical import path into the doc.go file again.  I seem to recall that this created much pain and suffering with gopkg.in and such... but I can see that mixed imports are going to be painful for some.

**Comment by sbinet** [comment]
`v1.4.0` is `vgo` compatible :)
thanks.
